src/tilemap/mod.rs

```diff
@@ -0,0 +1,52 @@
+pub mod farmable;
+
+pub use self::farmable::Farmable;
+use crate::config::TileProperty;
+use crate::phys::collision;
+use crate::{graphics, Iso2, Vec2};
+use hecs::{EntityBuilder, World};
+use ncollide2d::shape::Cuboid;
+use std::collections::HashMap;
+
+const TILE_SIZE: usize = 16;
+
+pub struct Tile {}
+
+//static\image
+pub fn new_tilemap(tilemap: &str, tile_prop: &HashMap<String, TileProperty>, world: &mut World) {
+    let mut tile_builder = EntityBuilder::new();
+    let default = TileProperty::default();
+
+    tilemap.split_whitespace().enumerate().for_each(|(y, row)| {
+        row.split(',').enumerate().for_each(|(x, tile)| {
+            let tile_details = tile_prop.get(tile).unwrap_or(&default);
+
+            tile_builder
+                .add(graphics::Appearance {
+                    kind: graphics::AppearanceKind::image("dirt"), //tile_details.image.clone()
+                    alignment: graphics::Alignment::Center,
+                    z_offset: -1000.0,
+                    ..Default::default()
+                })
+                .add(Iso2::translation(
+                    (TILE_SIZE / 2 + TILE_SIZE * x) as f32,
+                    (TILE_SIZE / 2 + TILE_SIZE * y) as f32,
+                ))
+                .add(Tile {});
```

Comment:
Why is it important or why would it be useful in the future to mark an entity as a Tile?
Can't we offer a less specific API through the ECS?

Reply:
Even if we do somehow find some reason to mark something as a tile in the future, that may never be the case and until then, this component is just bloat.